complex/ft_comp.c

```diff
@@ -50,6 +50,24 @@ static size_t comp_entry_size[] = {
 };
 */
 
+static int ft_open_cntrs(void)
+{
+	int ret;
+
+	opts.comp_method = test_info.cq_wait_obj;
```

Comment:
I looked into it a bit and realized that the config file enum takes the actual FI_...enums while the common/shared.c opening of the counters uses fabtest-defined FT_...enums which is probably causing the issue. I can look into it. Thanks for catching it!